README.md

```diff
@@ -46,7 +53,7 @@ MMODA is developed joinly by [UNIGE Department of Astronomy](https://www.unige.c
 
 * Added [Gravitatitional Wave analysis](https://www.astro.unige.ch/mmoda/help/mmoda/gravitational-wave-analysis) based on [GWOSC](https://www.gw-openscience.org/) analysis threads.
 * Added visible and infrared photometry from [DESI LegacySurvey](https://www.astro.unige.ch/mmoda/help/mmoda/legacy-survey) based on [NOIRLab](https://datalab.noirlab.edu/ls/dataAccess.php) TAP service.
-* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/).
+* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/). This allows the user to experience the potential of using a Jupyter notebook in a pre-built environment.
```

Comment:
the seconds sentence largely duplicates the first one. Also we wanted to keep the bullet points roughly same weight. I would remove the second sentence.

Reply:
technically, you are right, but I am not sure most people know that the second sentence rephrases the first one. I will not make a big point out of this, though.

Reply:
```suggestion
* Added possibility to open the MMODA results in a JupyterLab analysis environment at [RenkuLab](https://renkulab.io/). 
```